Frontend/ui-common/src/Services/Backend.purs

```diff
@@ -0,0 +1,174 @@
+module Services.Common.Backend where
```

Comment:
Common Backend file is not required. Keep these functions in Driver and customer folders